tst/com/amazon/ion/benchmark/testSchemaContainsElement.isl

```diff
@@ -0,0 +1,6 @@
+type::{
+    name: StructWithElement,
+    type: struct,
+    element: { type: timestamp, timestamp_precision: range::[year, day]},
+    container_length: 4,
```

Comment:
Do we have a file that covers the case where `element` is specified but `container_length` is not?

Reply:
> Do we have a file that covers the case where `element` is specified but `container_length` is not?

The ISL file which specifies `element` without specifying `container_length` will be added in the following commit.